File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        max_sum=0; current_sum=0;
        for i in range(len(accounts)):
            for j in range(len(accounts[i])):
                current_sum+=accounts[i][j]
            max_sum=max(max_sum,current_sum);
            current_sum=0
        return max_sum

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int max=Integer.MIN_VALUE, sum=0;
        for(int i=0;i<accounts.length;i++){
            for(int j=0;j<accounts[i].length;j++){
                sum+=accounts[i][j];
            }
            max=Math.max(sum,max);
            sum=0;
        }
        return max;
    }
}